sha-1 is only used in the build script, which is executed before the crate is
packaged for crates.io and which is not included in the crates.io package.
Index: pest-meta/Cargo.toml
===================================================================
--- pest-meta.orig/Cargo.toml
+++ pest-meta/Cargo.toml
@@ -29,3 +29,0 @@ version = "1.0"
-[build-dependencies.sha1]
-version = "0.10"
-default-features = false
